pkg/cmd/util/net.go

```diff
@@ -22,6 +22,69 @@ func TryListen(hostPort string) (bool, error) {
 	return true, nil
 }
 
+// tcpKeepAliveListener sets TCP keep-alive timeouts on accepted
```

Comment:
maybe keep all the stuff from http/server.go in its own file with a comment on where it came from?